src/System.ComponentModel.Primitives/src/System/ComponentModel/Component.cs

```diff
@@ -0,0 +1,257 @@
+//------------------------------------------------------------------------------
+// <copyright file="Component.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>                                                                
+//------------------------------------------------------------------------------
```

Comment:
I believe you should replace all comments of this sort with

// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.